test/model_simulation/simulate_ODEs.jl

```diff
@@ -37,25 +40,27 @@ let
         (k7, k8), ∅ ↔ X8
     end
 
-    for factor in [1e-1, 1e0, 1e1, 1e2, 1e3]
-        u0 = factor * rand(rng, length(get_unknowns(known_equilibrium)))
-        p = 0.01 .+ factor * rand(rng, length(get_ps(known_equilibrium)))
-        prob = ODEProblem(known_equilibrium, u0, (0.0, 1000000.0), p)
-        sol = solve(prob, Rosenbrock23())
-        @test abs.(sol.u[end][1] / sol.u[end][2] - p[2] / p[1]) < 10000 * eps()
-        @test abs.(sol.u[end][3] * sol.u[end][4] / sol.u[end][5] - p[4] / p[3]) <
-              10000 * eps()
-        @test abs.((sol.u[end][6]^2 / factorial(2)) / (sol.u[end][7]^3 / factorial(3)) -
-                   p[6] / p[5]) < 1e-8
-        @test abs.(sol.u[end][8] - p[7] / p[8]) < 10000 * eps()
+    for factor in [1e-1, 1e0, 1e1]
+        u0 = rnd_u0(known_equilibrium, rng; factor)    
+        p = rnd_ps(known_equilibrium, rng; factor, min = 0.1)
+        prob = ODEProblem(known_equilibrium, u0, (0.0, 100000.0), p)
+        sol = solve(prob, Vern7(); abstol = 1e-12, reltol = 1e-12)
+
+        @test sol[:X1][end] / sol[:X2][end] ≈ prob.ps[:k2] / prob.ps[:k1] atol=1e-8
+        @test sol[:X3][end] * sol[:X4][end] / sol[:X5][end] ≈ prob.ps[:k4] / prob.ps[:k3] atol=1e-8
+        @test (sol[:X6][end]^2 / factorial(2)) / (sol[:X7][end]^3 / factorial(3)) ≈ prob.ps[:k6] / prob.ps[:k5] atol=1e-8
+        @test sol[:X8][end] ≈ prob.ps[:k7] / prob.ps[:k8] atol=1e-8
     end
 end
 
-### Compares to Known ODE Function ###
-
-identical_networks_1 = Vector{Pair}()
-
+# Compares simulations generated through Catalyst with those generated by manually created functions.
 let
+    # Manually declares ODEs to compare Catalyst-generated simulations to.
+    catalyst_networks = []
+    manual_networks = []
+    u0_syms = []
+    ps_syms = []
```

Comment:
Many cases have had similar reameks now. One  vector to store one set of networks, one for another. Next I provide the list of the order the parameters and variables appears in in the manually created networks. These can be used to generate the correct u0/ps vectors from the randomly generated maps.